devops/aws/create-ec2-instance.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
This just looks like a leftover test script, I don't think it should be committed?

Reply:
yes I thought I had removed it already :)

Reply:
removed